src/test/java/org/mongojack/TestDBUpdateSerialization.java

```diff
@@ -153,6 +156,27 @@ public void testSimpleMap() {
         assertThat(coll.findOneById("id").map, equalTo(map));
     }
 
+    // Test to detect presence of issue #98
```

Comment:
please put a link to https://github.com/devbliss/mongojack/issues/98 instead of just saying "issue #98". this repo has lived in different places and may move again soon, so saying #98 isn't sufficiently clear